### DISCUSSION TITLE
貌似alist和rutorrent冲突

### DISCUSSION THREAD

**Discussion by WXpiero** [AlistGo/alist#637] [General]
问题一：在安装有rtinst1.8.17的ubuntu16.04盒子上用一键脚本安装alist后，只要alist不明原因自动重启一次，rutorrent就不工作了。有时候表现又是rutorrent界面还能用，但是filemanager插件无法访问目录。
只有停止alist服务以后，还原rutorrent备份配置才能恢复。。。
但是在装有swizzin的Debian11盒子上暂时没有此问题
问题二：然后明明vsftpd是有被rtinst设置登录账户的，但是添加ftp账号还是会显示非匿名账户必须要加密访问。
抱歉我还无法提供日志，希望大佬有兴趣关注一下这个问题。。。

**Comment by xhofe** [comment]
- alist除了默认会占用5244端口外，不会与其他软件产生冲突，若为端口原因，你可以修改默认端口
- ftp使用https://github.com/jlaffaye/ftp 你或许可以去这里找到解决方法

**Comment by auto8624** [comment]
怎么修改默认端口啊？
